### DISCUSSION TITLE
2007/12/28/labview-%E5%B7%A5%E7%A8%8B%E8%B5%84%E6%BA%90%E7%AE%A1%E7%90%86%E5%99%A8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#199] [Announcements]
# 2007/12/28/labview-%E5%B7%A5%E7%A8%8B%E8%B5%84%E6%BA%90%E7%AE%A1%E7%90%86%E5%99%A8

从 8.0 开始，LabVIEW 增加了一个工程资源管理器功能。LabVIEW 工程资源管理器就是一个可以方便查看、调整程序系统结构的工作区。与 VC, VB 等语言中的 project, workspace 相类似。Project 的出现使得 LabVIEW 对于大项目的管理更加方便。

https://blog.qizhen.xyz/2007/12/28/labview-%E5%B7%A5%E7%A8%8B%E8%B5%84%E6%BA%90%E7%AE%A1%E7%90%86%E5%99%A8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

arrowpig
2007年12月28日 21:01 编辑
LabVIEW8里面的Project View我觉得真的不错。可是最最受不了的就是VI和Library的互相依赖。VI记住了自己属于哪个Library，万一Crack了Library文件，真麻烦！
